modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -47,6 +53,34 @@ def service_information
       end
 
       def current_mailing_address
+        if address_is_military?
+          handle_military_address
+        else
+          handle_domestic_or_international_address
+        end
+      end
+
+      def address_is_military?
+        city = @data.dig(:veteranIdentification, :mailingAddress, :city)&.strip&.upcase
+        MILITARY_CITY_CODES.include?(city)
```

Comment:
I'm only checking for one value since we validate the combination works to get to this point it would need to be a valid city/state combination so if one is good the other is as well.  

If I am overlooking anything there feel free to let me know and I can adjust anything I just did not think a check for `state` as well added anything

Reply:
Can we share the logic, so we aren't duplicating code?

Reply:
@tycol7 Good idea [8b0e2fd](https://github.com/department-of-veterans-affairs/vets-api/pull/18376/commits/8b0e2fd5767761717100cee1d882763ff143fc6c)